core/modules/dashboard/tests/dashboard.test

```diff
@@ -111,6 +111,9 @@ class DashboardTest extends BackdropWebTestCase {
     // The manage user accounts link is present.
     $this->assertLink('Manage user accounts', 0, 'The Manage user accounts link was found.');
 
+    // The backdrop-news block was found.
+    $this->assertText('Backdrop News', 'Backdrop News block found.');
```

Comment:
If the feed URL was in settings, we could have a bit more tests, like 
- load the URL in the test, check its contents, and verify that the block displays the feed contents
- change the saved URL in settings, then test to see what the block displays (the proper "No news at this time", or if it blows up the site if the feed is not found for example)

Maybe not absolutely necessary, but would harden our testing a bit more.

Reply:
> load the URL in the test, check its contents, and verify that the block displays the feed contents

We can do this with a hard-coded URL, too. :)

> change the saved URL in settings, then test to see what the block displays

Hm, this is probably something we should test manually before release, but since it's not something anyone running backdrop could do (yet) I don't see why we need a test for it (yet). If we ever make it possible to change the feed URL we can add that test to go with that feature at the time we add it.